<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailConfigure</class>
 <widget class="QWidget" name="EmailConfigure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="GBox_MasterEmail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>380</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title">
    <string>主机邮箱配置</string>
   </property>
   <widget class="QLabel" name="Lab_MasterEmailAddress">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>20</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>主机邮箱:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEdit_MasterEmailAddress">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>19</y>
      <width>260</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Lab_MasterEmailPasswd">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>50</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>主机邮箱密码: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEdit_MasterEmailPasswd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>49</y>
      <width>260</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Lab_MasterEmailName">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>80</y>
      <width>65</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>邮件发送人: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEdit_MasterEmailName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>79</y>
      <width>260</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Lab_MasterEmailType">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>110</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>主机邮箱类型: </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBox_EmailOutlook">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>109</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Outlook</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBox_EmailTencent">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>109</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tencent</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GBox_RecvEmail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>380</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>收件人配置</string>
   </property>
   <widget class="QLabel" name="Lab_RecvEmailAddress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>65</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>收件人邮箱:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEdit_RecvEmailAddress">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>260</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_AddEmail">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>19</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>＋</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_DelEmail">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>48</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComBox_RecvEmailList">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>49</y>
      <width>260</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Lab_RecvEmailList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>65</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>收件人列表:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBox_FirstNotice">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>初次通知</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Btn_Enter">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lab_Tips">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>[提示]：当前版本并没有校验纠错功能，</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
